docs/pages/contracts/v4/guides/manage-liquidity.mdx

```diff
@@ -0,0 +1,280 @@
+import { Callout } from 'vocs/components'
+
+# Manage liquidity for CL and Bin pool
+
+We'll utilize the periphery contracts from [v4-periphery](https://github.com/pancakeswap/pancake-v4-periphery) to manage liqudity in v4.
+
+The topic would include:
+
+1. Permit2 integration
+2. PositionManager execution flow
+3. Type of cl pool liquidity actions (`CLPositionManager`)
+4. Type of bin pool liquidity actions (`BinPositionManager`)
+5. Type of settlement actions (applicable for both pool type)
+
+## 1. Permit2 integration
+Both [CLPositionManager](https://github.com/pancakeswap/pancake-v4-periphery/blob/main/src/pool-cl/CLPositionManager.sol) and [BinPositionManager](https://github.com/pancakeswap/pancake-v4-periphery/blob/main/src/pool-bin/BinPositionManager.sol) integrates with [Permit2](/contracts/permit2)
+
+**Frontend**
+
+For the frontend, this means that any operation involving token transfers from the user requires the user to sign a permit message and include that signature in the multiCall. 
+
+Example:
+
+```solidity
+bytes[] memory calls = new bytes[](2);
+
+// signature from user
+calls[0] = abi.encodeWithSelector(IAllowanceTransfer.Permit.selector, owner, PermitSingle permitSIngle, bytes signature);
+// modifyLiquidites call - explained in next section
+calls[1] = abi.encodeWithSelector(IPositionManager.modifyLiquidities.selector, abi.encode(...));
+
+positonManager.multiCall(...)
+```
+
+**Solidity**
```

Comment:
also i guess there should be one empty line to separate header and body otherwise it renders some how like:

![CleanShot 2024-09-02 at 10 19 27@2x](https://github.com/user-attachments/assets/ead87b93-9f5e-4373-9d48-3a0997b8d891)